app/build.gradle

```diff
@@ -5,12 +5,6 @@ def keyStoreDefined = project.hasProperty('signingStoreLocation') &&
         project.hasProperty('signingKeyAlias') &&
         project.hasProperty('signingKeyPassword')
 
-repositories {
-    maven {
-        url 'https://jitpack.io'
-    }
-}
-
```

Comment:
What's the goal of moving jitpack to the root `build.gradle`?
I heard issues about jitpack repo and security. Any word about it?

Reply:
It just keeps everything together.

Reply:
ok